docs/_future/manifest.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Manifest
+lang: en
+slug: manifest
+order: 2
+layout: future
+---
+
+
+<div class="section-content">
+
+Your project should contain a `manifest.json` file that defines your app's manifest. This is where you'll configure your application name and scopes, declare the functions your app will use, and more.
+Refer to the <a href="https://api.slack.com/reference/manifests" target="_blank">App Manifest documentation</a> to learn
+about the available manifest configurations.
+
+Notably, the App Manifest informs Slack of the definitions for:
+
+* [Functions](/bolt-python/future/concepts#manifest-functions)
+* [Workflows](/bolt-python/future/concepts#manifest-workflows)
+
+`manifest.json` is located at the top level of <a href="https://github.com/slack-samples/bolt-python-starter-template" target="_blank">our example projects</a>, along with a `triggers` folder containing
```

Comment:
Does the link to the template need to also link to the `future` branch?

Reply:
Very good catch 💯